src/platform/forms/sass/_m-schemaform.scss

```diff
@@ -474,6 +474,15 @@ legend.schemaform-label.schemaform-file-label {
   outline: none;
 }
 
+.form-review-panel-page-header {
+  padding-right: 1rem;
+  line-height: 2.5rem;
```

Comment:
I think we need to add `padding-top: 1rem` here as well because there's padding/margin on the edit button, so now the header sits a little tiny higher than the button.

Reply:
**Before:**
<img width="450" alt="Screenshot 2023-12-20 at 12 31 11 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/117098918/89ed9dd0-b758-4094-8967-0a39bbb1592d">

**After:**
<img width="621" alt="Screenshot 2023-12-20 at 12 32 15 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/117098918/7eb499c2-ea23-44f4-a172-47d9a0ca8596">

Reply:
oh, good call! I'll get that added

Reply:
Done